packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -240,6 +241,26 @@ extension TextFromString on String {
   }
 }
 
+/// Forces garbage collection by aggressive memory allocation.
+Future<void> _forceGC() async {
```

Comment:
isn't this duplicated from the leak tracking code?

Reply:
Do you mean the package leak_tracker. Yes, it contains the code, but internally. I do not think we want to publish it in API.